internal/cli/atlas/privateendpoints/aws/interfaces/describe.go

```diff
@@ -64,6 +65,13 @@ func DescribeBuilder() *cobra.Command {
 		Aliases: []string{"get"},
 		Args:    require.ExactArgs(1),
 		Short:   "Return a specific AWS private endpoint interface for your project.",
+		Annotations: map[string]string{
+			"args":           "endpointId",
+			"requiredArgs":   "endpointId",
+			"endpointIdDesc": "Unique alphanumeric string that identifies the private endpoint.",
+		},
+		Example: fmt.Sprintf(`  This example uses the profile named "myprofile" for accessing Atlas.
+  $ %s atlas privateendpoints aws interfaces describe vpce-svc-0123456789abcdefg --endpointServiceId 0123456789abcdefghijklmn -P myprofile`, cli.ExampleAtlasEntryPoint()),
```

Comment:
```suggestion
  $ %s privateendpoints aws interfaces describe vpce-svc-0123456789abcdefg --endpointServiceId 0123456789abcdefghijklmn -P myprofile`, cli.ExampleAtlasEntryPoint()),
```

Reply:
we need to remove the hardcoded "atlas" entry here as this will be set by `cli.ExampleAtlasEntryPoint()`, also make sure to run gen-docs after updating this.